### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jack829**
Please test this out. 
I think I ended up finding  and following the correct upgrade path but I jumped a few versions of expo and expo SDK at once so there might be some things that arose from that.

Expo absorbed create-react-native-app hence the changes in the commands.

### CODE REVIEW

**Comment by dblanchard13** [comment]
Yo did you run end up poking around in the app after this upgrade? Looks like the bottom tab icons aren't showing up:
screen shot 2018-10-03 at 10 04 48 pm

**Comment by jack829** [comment]
😬 ill sort er out today

**Comment by jack829** [comment]
Alright, give this one more whirl

File: src/screens/Login.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Login extends Component {
 
     }
     catch (err) {
```
Comment (dblanchard13) [inline src/screens/Login.js]: Can you switch to using the global `log` method and can you also prepend a message about where this error is being thrown from (ie `Error in login`)

File: src/GlobalUtils.js [hunk 2]
```diff
@@ -1,2 +1,26 @@
 
 global.log = console.log.bind(console, '\n-------------------\n\n')
+
+global.filterAndFormatIOSContactsInUnitedStates = contacts => (
```
Comment (jack829) [inline src/GlobalUtils.js]: I would argue to put this as a helper that is not attached to the global scope but is rather imported where needed. Just to reduce global scope pollution. There is a `helpers` folder where I would just add a `filterAndFormatIOSUSContacts.js`

File: src/GlobalUtils.js [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+  contacts.reduce((numbers, contact) => {
+    if ( contact && contact.phoneNumbers instanceof Array ) {
+      let usPhoneNumbers = contact.phoneNumbers.reduce((numbers, { countryCode, digits }) => {
```
Comment (jack829) [inline src/GlobalUtils.js]: NIT should be `const`

**Pull request merged**